recipe/meta.yaml

```diff
@@ -74,10 +77,12 @@ outputs:
         - automake  # [not win]
       host:
         - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+        # we need headers from cuda-nvcc-dev_win-64, e.g. crt/host_defines.h
+        - {{ compiler('cuda') }}                    # [(cuda_compiler_version or "").startswith("12") and win]
```

Comment:
Have found doing this on Windows is reasonably effective

```batch
set "INCLUDE=%INCLUDE%;%BUILD_PREFIX%\Library\include"
```

Ofc it is a better if we don't have to. However these headers were not broken out until a later version of the CTK. So this may be a reasonable workaround for now

Reply:
> However these headers were not broken out from the compiler until a later version of the CTK.

Do you know which version? Then I'll leave a comment for when to drop it. I prefer that over accumulating hacks in the build script, because those are less visible than `meta.yaml`.